src/kv/key.rs

```diff
@@ -117,6 +118,12 @@ impl AsRef<Key> for Key {
     }
 }
 
+impl AsRef<Key> for Vec<u8> {
```

Comment:
Where is this one used?

Reply:
Here: https://github.com/tikv/client-rust/pull/109/files#diff-d01850033b0d2fc084df1829a7789724R299

Reply:
Hmm, ok right. So I think there is a bit of an issue that we have to convert back and forth between our `Key` abstraction and the underlying `Vec<u8>`, in this case because we are using the protobuf `Mutation` rather than our own kind of mutation. Not sure what would be better, but worth thinking about for the future